tsinfer/inference.py

```diff
@@ -253,16 +267,17 @@ def infer(
 def generate_ancestors(
     sample_data,
     *,
-    num_threads=0,
     path=None,
     exclude_positions=None,
+    # Performance-related parameters below
```

Comment:
I don't think this comment really helps - there's only one performance related parameter (num_threads) which is pretty self-explanatory.

Reply:
Fair enough - I was just trying to sort out a standardised order for listing parameters. Given that these are keyword-only, it's only to make things logical. And for the undocumented ones it doesn't really matter, I guess (see below).